exampleCourse/questions/demoAutograderPandas/question.html

```diff
@@ -0,0 +1,25 @@
+<pl-question-panel>
+  <p>
+    This question uses the built-in methods in <code>code_feedback.py</code> to check Pandas dataframes for correctness <i>(special thanks to Wade Fagen-Ulmschneider!)</i>.  By default, rows can be in any order as long as the correct data is present.  There is also an option to display the student input in the feedback tab (<code>display_input</code>, default false).
```

Comment:
`<code>display_input</code>` -> `<code>display-input</code>`

Reply:
This is a python keyword arg, I will update the text to be more clear.